src/BookResourceStore.ts

```diff
@@ -0,0 +1,99 @@
+import Manifest from "Manifest";
+
+export interface IBookStore {
+  href: string;
+  data: Blob;
+}
+
+// IndexDB store for Book resources
+// Only works on browsers that support IndexedDB.
+export default class BookResourceStore {
+  private readonly db: IDBDatabase;
+
+  public constructor(db: IDBDatabase) {
+    this.db = db;
+  }
+
+  public static createBookResourceStore(): Promise<BookResourceStore> {
+    if (!("indexedDB" in window)) {
+      //TODO: a polyfill to handle lack of IndexedDB
```

Comment:
Can you make sure a ticket gets create to handle polyfilling for browsers that don't have IndexedDB?